modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -927,6 +927,7 @@ exports.exportSearchResults = (req, resp) ->
 	serverUtilityFunctions.ensureExists exportedSearchResults, 0o0744, (err) ->
 		if err?
 			console.log "Can't find or create exportedSearchResults folder: " + exportedSearchResults
+			console.error err
```

Comment:
*outside pr scope*: I wanted this to print the actual error. I initial had a weird local setup that was producing and an error and I added this line to debug the issue.